premis_event_service/models.py

```diff
@@ -36,9 +36,6 @@ class Agent(models.Model):
         help_text="Optional note about agent."
     )
 
-    def __unicode__(self):
-        return self.agent_name
-
```

Comment:
The purpose of [`__str__`](https://docs.djangoproject.com/en/1.11/ref/models/instances/#str) is to define what the value is when you call `str` with an instance of this model. In this case, I don't think that would default to `agent_name`, and if we don't define the method, it will use something else.